admin_guide/managing_networking.adoc

```diff
@@ -1081,6 +1081,9 @@ other projects. To isolate a project, opt in to isolation by configuring the
 Namespace object, then create a `NetworkPolicy` object to indicate the allowed
 incoming connections.
 
+Project administrators can create and delete NetworkPolicy objects within their
+own project.
```

Comment:
Text is good but maybe move it further down, to the "Create the `NetworkPolicy` objects" item?